qemu/tests/cfg/thin_provisioning_image.cfg

```diff
@@ -0,0 +1,49 @@
+- thin_provisioning_image:
```

Comment:
what's the different between thin_provisioning and thin_provisioning_image? 
it's better the make the name more clear, then we can get the meaning from the case name